vulnfeeds/git/versions.go

```diff
@@ -33,44 +33,69 @@ func fuzzyVersionToCommit(normalizedVersion string, repo string, normalizedTags
 	}
 	// We may now have one or more tags to further examine for a best choice.
 	if len(candidateTags) == 0 {
-		return gc, false
+		return ac, false
 	}
 	if len(candidateTags) == 1 {
-		return cves.GitCommit{
-			Repo:   repo,
-			Commit: normalizedTags[candidateTags[0]].Commit,
-		}, true
+		ac.SetRepo(repo)
+		switch commitType {
```

Comment:
Can these also be changed to use the enum?

Reply:
Yes, missed them. Done in https://github.com/google/osv.dev/pull/1510/commits/0a69ddfa848b2722f6e019d5c975877d9015ff32